app/services/analytics.ts

```diff
@@ -0,0 +1,74 @@
+import { action } from '@ember-decorators/object';
+import Service from '@ember/service';
+import Ember from 'ember';
+import config from 'ember-get-config';
+
+export default class Analytics extends Service {
+    metrics = Ember.inject.service();
+    session = Ember.inject.service();
+
+    @action
+    click(category, label, extraInfo) {
+        let extra = extraInfo;
+        if (extra && typeof extra !== 'string') {
+            // This is to remove the event object when used with onclick
+            extra = null;
+        }
+        Ember.get(this, 'metrics')
```

Comment:
`this.get(...)` or `get(this, ...)` with `import { get } from '@ember/object'`, but prefer not using `Ember.*`